frontend/shared/startup.js

```diff
@@ -29,6 +29,26 @@ const getCookie = function(name) {
     sessionid = getCookie("sessionid"),
     csrfSafeMethod = method => /^(GET|HEAD|OPTIONS|TRACE)$/.test(method);
 
+var intervalID;
```

Comment:
could you move this code into a separate file,  for example, see the as an example. That way all the variable closures will a single place

```javascript
import sidebarStartup from "./utils/sidebarStartup";
```